index.html

```diff
@@ -17,12 +17,11 @@ <h1 id="top">
       W3C Workshop on Web &amp; Virtual Reality
     </h1>
     <p>
-      October 2016, 19-20, San Jose, CA, USA
+      October 19-20, 2016; San Jose, CA, USA
```

Comment:
it might be useful to include the venue below here so people don't have to dig through to the Logistics section. thoughts?